components/tabs/demo/tabs.html

```diff
@@ -35,35 +36,17 @@ <h2>Tabs</h2>
 			</d2l-demo-snippet>
 
 			<h2>Tabs (paired d2l-tab with d2l-panel)</h2>
+			<div>This format is still a WIP. Please do not use yet.</div>
 
 			<d2l-demo-snippet>
-				<div>This format is not yet functional. Please do not use yet.</div>
 				<template>
 					<d2l-tabs>
 						<d2l-tab id="all" text="All" slot="tabs"></d2l-tab>
```

Comment:
This is the order I would typically expect. But I would be curious if there is any adverse affect on the logic if the panels come first in the DOM?

Reply:
Gave it a quick test and it seems to function the same either way